lib/galaxy/jobs/runners/drmaa.py

```diff
@@ -68,6 +68,9 @@ def __init__( self, app, nworkers, **kwargs ):
                                 (exc.__class__.__name__, str(exc)))
         from pulsar.managers.util.drmaa import DrmaaSessionFactory
 
+        # make the drmaa library also available to subclasses
+        self.drmaa = drmaa
```

Comment:
Is it not possible to import drmaa in the subclasses?

Reply:
There were 2 points that made me think that this is a good solution: 
- the init function of DRMAAJobRunner takes quite some effort to import drmaa see: line 56-68 (eg. setting/using library paths) and I wanted to reuse this .. 
- there is a global drmaa variable which led to some naming confusion in the subclass .. if I remember correctly. I guess this was because the UnivaRunner calls the superclass init function and which does the import. 

Maybe the real question is if the import should be done in the init function?

Reply:
In the same way self.drmaa_job_states on the following lines is kind of ugly which is used then in the SLURMRunner. I basically have drawn my inspiration from this.

Reply:
Ah right, this is to support the `$DRMAA_LIBRARY_PATH` environment variable. Making the module itself an instance attribute seems a bit dirty to me, though, so I'm going to attempt to do this a different way and make a PR against your branch if I get something working that I like.